pyspecProcScripts/integrate_limits.py

```diff
@@ -17,9 +17,11 @@ def integrate_limits(s, axis="t2", fwhm=100, fl=None):
     signal_E /= signal_E.data.max()
     filter_width = abs(signal_E-1/sqrt(2)).argmin('sigma').item()
     if fl is not None:
+        fl.push_marker()
         fl.next('signal Energy')
         fl.plot(signal_E, human_units=False)
         fl.plot(signal_E['sigma':(filter_width,filter_width+1e-6)],'o', human_units=False)
+        fl.pop_marker()
```

Comment:
note use of push an pop -- in general, when you have a function make diagnostic plots, it's good practice to call push at before the function starts any plotting, and to call pop at the end.  that way,  no matter what happens outside the function, you don't end up putting stuff on the wrong plot.